### ISSUE TITLE
Request api call not triggered in network pannel

### ISSUE THREAD

**Issue by Bizotics-Dev** [angular/angular#41241]
Angular version 9.0
If we are fetching same api call then it will triggered only once in network pannel second time it not showing in network
below example:

this.getTimeCollections().then(data=>{console.log(data)});

//in service.ts

getTimeCollections():Promise{

    const url = `${this.publicUrl}/time_collections`;

    return this.httpClient.get(url).toPromise();
}

response is coming but it's old data. We tried with using observable also but it is not helpful.
It's not consistent  but sometime it's happening.
Thanks in advance

**Comment by gkalpak** [comment]
There is nothing in `HttpClient` that would serve a cached response without making an actual request. It sounds like something else in your app is causing this.

I suggest reaching out in our of our general support forums (such [StackOverflow](https://stackoverflow.com/) or [Discord](https://discord.com/invite/angular)).

<sub>If you are wondering why we don't resolve support issues via the issue tracker, please [check out this explanation](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-got-a-question-or-problem).</sub>

##
If you have reasons to think this is a bug in Angular, please open a new issue providing a minimal reproduction.
